src/store/mutations/mutations-model.ts

```diff
@@ -29,8 +29,8 @@ export const mutateAuthRoles = (state: any, authRoles: Array<string>) => {
   state.stateModel.tombstone.authRoles = authRoles
 }
 
-export const mutateUserEmail = (state: any, userEmail: string) => {
-  state.stateModel.tombstone.userEmail = userEmail
+export const mutateUserInfo = (state: any, userInfo: any) => {
+  state.stateModel.tombstone.userInfo = userInfo
```

Comment:
I like this approach as well! Easier to get and set these sections as a whole, which is what we seem to do often.

Reply:
Yeah, though it does depend what the object is. In this case I don't care, hence type `any`.